packages/editor/src/components/editor-help/add-blocks.native.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+import { HelpDetailBodyText, HelpDetailImage } from './view-sections.native';
```

Comment:
I believe we shouldn't need to append the `.native` suffix ourself and generally rely upon Metro to manage this. Also, this extra suffix occurs a few more times in this PR, but I did not call them out individually to avoid creating unnecessary noise. We should likely update those locations as well. 
```suggestion
import { HelpDetailBodyText, HelpDetailImage } from './view-sections';
```

Reply:
Ah good point. Updated in b2b136317383f3b251ce0d39977bdad791306bbe